spec/cypress/integration/post/create_post.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Create post', () => {
+    beforeEach(() => {
+      cy.scenario('sign_in');
+      cy.visit('/');
+      cy.get('[data-id="link-to-login"]').click();
+  
+      cy.url().should('include', '/sign_in');
+  
+      cy.get('[name="user[email]"]').type('user@example.com');
+      cy.get('[name="user[password]"]').type('password');
+  
+      cy.get('input.btn')
+        .contains('Log in')
+        .click();
```

Comment:
Lets move this to a `Cypress.Commands` 
* in `spec/cypress/support` remove `command.js` and make it a directory so `spec/cypress/support/commands/`
* Within that directory create a new file, called `user.js` so it'll be `spec/cypress/support/commands/user.js `
* Inside that `user.js` file you can add the following
```
Cypress.Commands.add('userLogin', (user = {}) => {
      const { email = 'user@example.com', password = 'password' } = user; // This will allow us to override the user's details in the future

      cy.visit('/');
      cy.get('[data-id="link-to-login"]').click();

      cy.url().should('include', '/sign_in');

      cy.get('[name="user[email]"]').type(email);
      cy.get('[name="user[password]"]').type(password);

      cy.get('input.btn')
        .contains('Log in')
        .click();
});
```

* Then back in this file `spec/cypress/integration/post/create_post.spec.js` you can just have something like
```
beforeEach(() => {
    cy.scenario('sign_in');
    cy.userLogin();
});
```

Reply:
If webpack can not find the new files, From inside the `spec/sypress/support/command` folder add a new file called `index.js`. Within that file add the following

```js
import './user';
```